data/json/monsters/cyborgs.json

```diff
@@ -33,7 +33,7 @@
       "SEES",
       "HEARS",
       "BASHES",
-      "ATTACKMON",
+      "PUSH_MON",
```

Comment:
Cyborgs.json shouldn't have Push_Mon

Reply:
If you're worried about them not going after zombies when they should be - they're members of the 'science' faction, which is actually a subset of zombies, so they shouldn't be in any event.